### PR TITLE
Issue #57: Support both dot and comma as decimal separator

### LINKED ISSUE #57: Support both dot and comma as decimal separator
As backend returns different price format for different locales (5,55 for de_DE and 5.55 for en_US) javascript which parses the price for filter navigation must support both.

### PR DESCRIPTION
**Pull request by tim-bezhashvyly**
Closes #57.

### CODE REVIEW

**Comment by tim-bezhashvyly** [comment]
Ready for the next review.

File: tests/JavaScript/FilterNavigationSpec.js [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+                expect(filterOptions.childNodes.length).toBe(1);
+
+                var filterOptionsList = filterOptions.childNodes[0];
+
+                expect(filterOptionsList.tagName).toBe('OL');
+                expect(filterOptionsList.className).toBe('filter-content filter-' + filterCode);
+
+                expect(filterOptionsList.childNodes.length).toBe(2);
+
+                Array.prototype.map.call(filterOptionsList.childNodes, function (filterOption, index) {
```
Comment (Vinai) [inline tests/JavaScript/FilterNavigationSpec.js]: Hmm, shouldn't it be possible to call `filterOptionsList.childNodes.map(...)`?

File: tests/JavaScript/FilterNavigationSpec.js [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
```
Comment (Vinai) [inline tests/JavaScript/FilterNavigationSpec.js]: Same as above - I think it should be possible to call `map` on the child nodes directly.

File: tests/JavaScript/FilterNavigationSpec.js [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
```
Comment (Vinai) [inline tests/JavaScript/FilterNavigationSpec.js]: And again. If I'm wrong about this please disregard.

**Pull request merged**